src/notebook/widget.ts

```diff
@@ -201,6 +200,7 @@ class NotebookWidget extends Widget {
       widget = new BaseCellWidget(cell);
     }
     widget.addClass(NB_CELL_CLASS);
+    widget.input.editor.addClass(NB_EDITOR_CLASS);
```

Comment:
I moved the addClass logic out of the factory so we preserve the tagging in subclasses.